src/components/CookieNotice/CookiePolicy.tsx

```diff
@@ -0,0 +1,101 @@
+import "./CookiePolicy.scss";
+import store from "store";
+import {ActionFactory} from "store/action";
+import Portal from "components/Portal/Portal";
+
+interface CookiePolicyProps {
+  scrumlrCookieName: string;
+  acceptFunction: () => void;
+  onClose: () => void;
+  show: boolean;
+}
+
+const CookiePolicy = ({scrumlrCookieName, acceptFunction, onClose, show}: CookiePolicyProps) => {
+  if (!show) {
+    return null;
+  }
+
+  const handleAccept = () => {
+    acceptFunction();
+    store.dispatch(ActionFactory.addCookie(scrumlrCookieName, true));
+  };
+
+  return (
+    <Portal onClose={onClose}>
+      <div className="cookie-policy">
+        <div className="cookie-policy__title">
+          <h3>Cookie Policy of scrumlr.io</h3>
+        </div>
+        <div className="cookie-policy__body">
+          <p>
```

Comment:
This text is rather lengthy and it might be exhausting to change something here. I would suggest to import the text as markdown and maybe you can research some different ways to do it? https://stackoverflow.com/questions/42928530/how-do-i-load-a-markdown-file-into-a-react-component suggests `marked` but maybe there are even simpler ways by now to do it in `create-react-app`. Dunno